### ISSUE TITLE
utf-8 support

### ISSUE THREAD

**Issue by Halama** [keboola/db-writer-mssql#24]
Extractor is breaking utf-8 characters even if column type is set `nvarchar`

![image](https://user-images.githubusercontent.com/903531/39700355-2f5fed72-51fd-11e8-8f3a-87ef305429be.png)

https://connection.keboola.com/admin/projects/2703/writers/keboola.wr-db-mssql-v2/292434707

**Comment by Halama** [comment]
https://support.microsoft.com/en-us/help/3136780/utf-8-encoding-support-for-the-bcp-utility-and-bulk-insert-transact-sq ?

**Comment by Halama** [comment]
Tak to padá na ` Output: -C not supported in this release. usage: bcp...`

```
bcp -v
BCP - Bulk Copy Program for Microsoft SQL Server.
Copyright (C) Microsoft Corporation. All Rights Reserved.
Version: 17.1.0000.1
```

Instalace podle https://docs.microsoft.com/cs-cz/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-2017#microsoft-odbc-driver-131-for-sql-server

**Comment by Halama** [comment]
Pokud to nastavím v db params tak `bcp` padá exit code `1`:
```
"db": {
      "driver": "mssql",
      "host": "mssql",
      "port": 1433,
      "database": "test",
      "user": "sa",
      "#password": "yourStrong(!)Password",
      "collation": "65001"
    },
```

**Comment by Halama** [comment]
@MiroCillik tušíš jestli se teda nějaká dá naimportovat utf-8?

**Comment by Halama** [comment]
https://keboola.zendesk.com/agent/tickets/8367

**Comment by Halama** [comment]
Fixed in #27
